### PR TITLE
using largelist instead of flatlist. fix #116

### LINKED ISSUE #116: Scroll offset does not adjust with component height
The loading interface appears in the drop-down, the loading icon disappears after receiving the data, and the occupied space also disappears, but the scroll offset has not been adjusted accordingly, but it does not affect the user experience much. (Probably a FlatList bug).

see PR #115

### PR DESCRIPTION
**Pull request by AmazingBeerBelly**
Using LargeList instead of FlatList to resolve the Android scroll animation issue and fix issue #116 (Scroll offset does not adjust with component height).

ios LargeList presentation:

Kapture 2020-05-26 at 16 59 27

Kapture 2020-05-26 at 17 36 22

android LargeList presentation:

### CODE REVIEW

File: src/pages/wallet/history.js [hunk 1]
```diff
@@ -231,12 +228,20 @@ const stateIcons = {
   Failed: <MaterialIcons name="error-outline" size={36} style={[{ color: '#E73934' }]} />,
 };
 
+class MyRefreshHeader extends RefreshHeader {
```
Comment (imstar15) [inline src/pages/wallet/history.js:231]: 需要改个名字，HistoryRefreshHeader。在另外一个文件里，写了两个class，应该不能通过eslint检查哦。你可以写成将class改成function

File: src/components/headers/header.history.refresh.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { ActivityIndicator } from 'react-native';
+import { RefreshHeader as Header } from 'react-native-spring-scrollview';
+
+export default class RefreshHeader extends Header {
+    static height = 50;
```
Comment (chrisli30) [inline src/components/headers/header.history.refresh.js:6]: 这个 hardcode的50是什么意思。我并没有看到 `RefreshHeader.height`的引用啊

Comment (chrisli30) [reply src/components/headers/header.history.refresh.js:6]: 如果是必须的，需要加注释

File: src/pages/wallet/history.js [hunk 3]
```diff
@@ -475,6 +466,21 @@ class History extends Component {
     navigation.navigate('Transaction', item);
   }
 
+  loadMoreData = () => {
+    const { isLoadMore, isRefreshing, listData } = this.state;
+    // In these cases, the operation of loading more should not be executed.
+    // 1. the list data is empty
+    // 2. It's loading more
+    // 3. When FlatList momentum scroll, the onEndReached function is called before.
```
Comment (chrisli30) [inline src/pages/wallet/history.js:474]: comments are good 👍

**Pull request merged**